api/src/main/resources/liquibase.xml

```diff
@@ -4,6 +4,309 @@
                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+    <changeSet author="slubwama" id="patientflags-1554735240623-1">
```

Comment:
@slubwama Add a comment tag to each of the changesets describing what each of them is doing

Reply:
@ssmusoke done